bundle/default-bundle/start.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 JAVA_OPTS="${JAVA_OPTS}"
+CUSTOM_CLASSPATH_FOLDER="${CUSTOM_CLASSPATH_FOLDER:-/opt/custom}"
```

Comment:
I thought we support `CUSTOM_CLASSPATH` and just use that variable and default it to: `/opt/custom/*` in case customers want to mount a single file instead of a directory.

Reply:
Removed env variable